packages/server/src/service/alloy.ts

```diff
@@ -67,5 +81,15 @@ export const isRegistered = async (voter: StateInfo): Promise<RegistrationStatus
     }
   }) as AxiosResponse<AlloyResponse>
 
-  return response.data.data.registration_status
+  // When not found Alloy API returns null, we default this to Not Found
+  return response.data.data.registration_status ?? 'Not Found'
+}
+
+const cacheIsRegistered = cache(
```

Comment:
Close -- let's create `isRegistered` (no cache) and `cacheIsRegistered`.  We only call the latter from testing.